packages/medusa/src/api/store/customers/query-config.ts

```diff
@@ -38,6 +38,10 @@ export const defaultStoreCustomerAddressFields = [
 
 export const retrieveAddressTransformQueryConfig = {
   defaults: defaultStoreCustomerAddressFields,
+  allowed: [
```

Comment:
todo: Can you add `validateAndTransformQuery` to the middlewares for the `GET /store/customers/me` endpoint? It is currently missing, so we can't query anything 😄

Reply:
it is present already

<img width="556" alt="Screenshot 2024-10-22 at 14 44 14" src="https://github.com/user-attachments/assets/404c8ef6-47ad-4e0b-afc7-33e23946d4bf">

Reply:
Oh my bad, but looking at the wrong middelware config